network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -59,7 +59,7 @@ <h2 class="h1-heading col-md-10">{{product.name}}</h2>
       <div id="product-research" class="mt-5">
         <div class="mb-5 d-flex flex-column flex-sm-row align-items-sm-center">
           <h3 class="h5-heading d-inline-block mb-2 mb-sm-0 mr-3">Our Research</h3>
-          <a href="/en/privacynotincluded/about" class="info-help">How to use this guide</a>
+          <a href="/privacynotincluded/about" class="info-help">How to use this guide</a>
```

Comment:
I wonder if there's a way to turn this into a `{{ pageurl about_page }}` instruction of some sort, so that we don't have to rely on this page actually being titled "About" in the cms...?

Reply:
Using built-in template tag `url` works!